src/Scenes/GameScene.js

```diff
@@ -45,6 +45,7 @@ export default class GameScene extends Phaser.Scene {
       frictionAir: 1,
       fixedRotation: true,
     });
+    lastRope.setInteractive({ useHandCursor: true });
```

Comment:
I think this fixes the conflict?

```suggestion
    this.ropeAnchor.setInteractive({ useHandCursor: true });
```

Reply:
Yeah should do